terminus-ui/selection-list/src/selection-list-panel/selection-list-trigger.directive.ts

```diff
@@ -73,6 +72,9 @@ export const TS_SELECTION_LIST_SCROLL_STRATEGY_FACTORY_PROVIDER = {
   useFactory: TS_SELECTION_LIST_SCROLL_STRATEGY_FACTORY,
 };
 
+// The max height of the select's overlay panel
+export const SELECTION_LIST_PANEL_MAX_HEIGHT = 256;
```

Comment:
Is it the max height regardless current window/screen size?

Reply:
yep, just like select and autocomplete.